### ISSUE TITLE
interfacial water orientational entropy 

### ISSUE THREAD

**Issue by jkalayan** [CCPBioSim/CodeEntropy#18]
The orientational entropy of water molecules at interfaces is currently calculated in POSEIDON, but is not modular and includes test code for theories that are no longer relevant. Instead, the useful parts of the code from POSEIDON need to be incorporated into the main CodeEntropy functions, once this is done we can remove POSEIDON from CodeEntropy.

The new implementation of water orientational entropy will include:
- Frame-by-frame detection of interfacial water molecules, these selections can subsequently be used to calculate the vibrational entropies of interfacial water molecules using existing CodeEntropy functions
- Identification of the coordination shell environment of each interfacial water using the RAD algorithm
- Identification of the hydrogen bonds donated to and accepted from each interfacial water molecule
- Calculation of the orientational entropy of each interfacial water molecule over all selected frames using coordination shell and hydrogen-bonding information.

At the moment, interfacial water molecules are defined as any water that is in the coordination shell of a molecule that is not a water molecule.

**Comment by jimboid** [comment]
Reminder to see branch 18-interfacial-water-orientational entropy for the starting point

**Comment by jkalayan** [comment]
This code in this branch will be moving to a separate importable package here: https://github.com/jkalayan/waterEntropy, this way the water specific code will be separated from the generalised code being developed here in CodeEntropy.
